mitodl/micromasters
# Created dialog for course team contact payment teaser

#### What are the relevant tickets?

Closes #2469 

#### What's this PR do?

Enables the 'Contact course team' link for unpaid users and shows them a message that they need to pay.

#### Where should the reviewer start?

DashboardPage.js

#### How should this be manually tested?

Set `MAILGUN_RECIPIENT_OVERRIDE` to your email, then check the behavior and appearance of the 'Contact course team' link on the dashboard with the following scenarios:

1. The user is enrolled & paid, and the course has a contact email
1. The user is enrolled but unpaid, and the course has a contact email
1. The course has no contact email

#### Screenshots (if appropriate)

![ss 2017-02-02 at 12 44 46](https://cloud.githubusercontent.com/assets/14932219/22572859/336f9f04-e974-11e6-9772-9a1485295a9d.png)
